src/features/src/changeSpeed.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import FFmpegFeature from '../FFmpegFeature';
+import Slider from '../../clui-ui-components/Slider'
+
+type ChangeSpeedConfig = {
+  SPEED: { value: number; [name: string]: any }
+  [name: string]: any
+}
+
+class ChangeSpeedFeature extends FFmpegFeature {
+  configuration: ChangeSpeedConfig
+  speed = 0;
+
+  constructor(configuration: ChangeSpeedConfig) {
+    super()
+    this.configuration = configuration
+    const { speedValue } = this.parseConfiguration()
+    this.speed = speedValue;
+    var speedStr = (1/(speedValue/100)).toString().substring(0, 4);
```

Comment:
Using `var` in prod code

Also this line is quite unreadable? what exactly are you doing here?